loading-kernel-module/README.md

```diff
@@ -0,0 +1,134 @@
+# Loading Kernel Module
+
+### Get the correct kernel-version
+
+We need the correct kernel version in order to build the kernel-module for the
+right kernel.
+
+Since after reboot, the kernel of the host will be the kernel RPM installed on
+the new image, aka, the `fedora-coreos:stable` image that we are using as
+the last layer, then the correct way to get the kernel version is by getting it
+from the image itself.
+```
+$ podman run -it fedora-coreos:stable rpm -qa | grep kernel
+kernel-modules-core-6.3.8-200.fc38.x86_64
+kernel-core-6.3.8-200.fc38.x86_64
+kernel-modules-6.3.8-200.fc38.x86_64
+kernel-6.3.8-200.fc38.x86_64
+```
+
+Let's export the kernel version.
+```
+export KERNEL_VERSION=6.3.8-200.fc38.x86_64
+```
+
+### Build the container image
+
+Now, we will build a simple container image that will contains a basic
+kernel-module in it.
+
+Since the kernel we want to build is a `fc38` kernel we will use `fedora:38` as
+a base image to build the kernel-module.
+
+```
+podman build --build-arg KERNEL_VERSION=${KERNEL_VERSION} -t quay.io/ybettan/fedora-coreos:kmm-kmod -f Containerfile
+podman push quay.io/ybettan/fedora-coreos:kmm-kmod
+```
+
+### Download the Fedora-CoreOS disk image
+
+Now, we are going to download the Fedora-CoreOS disk image for Qemu. We
+are going to use that disk in order to boot a VM from it later on this
+tutorial.
+
+The version of Fedora we are using for this VM doesn't really matter to much as we
+are going to reboot from the contianer we built anyway.
+
+```
+curl -L https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/36.20220716.3.1/x86_64/fedora-coreos-36.20220716.3.1-qemu.x86_64.qcow2.xz -o fedora-coreos-36.20220716.3.1-qemu.x86_64.qcow2.xz
```

Comment:
I think this duplicates https://docs.fedoraproject.org/en-US/fedora-coreos/provisioning-libvirt/#_launching_a_vm_instance which is better because it has instructions that will work over time instead of a hardcoded version.

Reply:
Updated to point to the fedora-coreos docs.